docs/open-telemetry-onboarding-resource-guide.md

```diff
@@ -0,0 +1,45 @@
+# Open Telemetry Onboarding Resource Guide
+
+Hi! If you’re looking at this document, this guide will provide you the resources as a newcomer to the Open Telemetry project. This guide will help you understand the Open Telemetry Project, its components, and specifically the C++ SDK.
+
+### Reading Resources
+
+* Medium [article](https://medium.com/opentelemetry/how-to-start-contributing-to-opentelemetry-b23991ad91f4) (October 2019) on how to start contributing to the Open Telemetry project.
+* Medium [article](https://medium.com/opentelemetry/opentelemetry-beyond-getting-started-5ac43cd0fe26) (January 2020) describing the overarching goals and use cases for Open Telemetry.
+
+### Relevant Documentation
+
+* Open Telemetry [Specification](https://github.com/open-telemetry/opentelemetry-specification)
+    * The Open Telemetry Specification describes the requirements and expectations of for all Open Telemetry implementations.
+
+* Read through the [Open Telemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) GitHub repository
+    * This repository has a lot of good information surrounding the Open Telemetry ecosystem. At the top of the **[readme](https://github.com/open-telemetry/opentelemetry-collector/blob/master/README.md)**, there are multiple links that give newcomers a good idea of what the project is about and how to get involved in it.
+* Read through the Open Telemetry Python documentation for its [API](https://opentelemetry-python.readthedocs.io/en/stable/api/api.html) and [SDK](https://opentelemetry-python.readthedocs.io/en/stable/sdk/sdk.html)
+    * The [API](https://github.com/open-telemetry/opentelemetry-java/blob/master/QUICKSTART.md) and [SDK](https://github.com/open-telemetry/opentelemetry-java/blob/master/QUICKSTART.md) documentation provides a lot of information on what the classes and their functions are used for. Since there is currently minimal documentation for C++, use the Python repository’s extensive documentation to learn more about how the API and SDK work.
```

Comment:
Link is to Java but no mention of Java in the text?

Reply:
I believe those should be linking to the API and SDK documentation for Python, thanks for catching that. Will fix it